configs/selfsup/beitv2/README.md

```diff
@@ -14,6 +14,8 @@ Masked image modeling (MIM) has demonstrated impressive results in self-supervis
 
 ## Models and Benchmarks
 
+You could download **VQ-KD** model from this [link](https://download.openmmlab.com/mmselfsup/1.x/target_generator_ckpt/vqkd_encoder.pth).
```

Comment:
Better tell users the ``vqkd`` target generator will be downloaded automatically, instead of downloading it manually.